aries_vcx/src/handlers/out_of_band/receiver.rs

```diff
@@ -51,7 +52,7 @@ impl OutOfBandReceiver {
                                 return Ok(Some(connection));
                             }
                         };
-                        if did_doc.resolve_service()? == service.resolve(pool_handle).await? {
+                        if did_doc.resolve_service()? == resolve_service(pool_handle, &service).await? {
```

Comment:
how about we rename `did_doc.resolve_service` to  `did_doc.get_service`? As that function is only trying to construct service locally.